pkg/common/utils/utils.go

```diff
@@ -142,6 +142,21 @@ func PodUnderCondition(pod *v1.Pod, condition *v1.PodCondition) bool {
 	return current != nil && current.Status == condition.Status && current.Reason == condition.Reason
 }
 
+// get namespace guaranteed resource from namespace annotation
+func GetNamespaceGuaranteedFromAnnotation(namespaceObj *v1.Namespace) *si.Resource {
+	// retrieve guaranteed resource info from annotations
+	namespaceGuaranteed := GetNameSpaceAnnotationValue(namespaceObj, constants.NamespaceGuaranteed)
+	var namespaceGuaranteedMap map[string]string
+	err := json.Unmarshal([]byte(namespaceGuaranteed), &namespaceGuaranteedMap)
+	if err != nil {
+		log.Logger().Warn("Unable to process namespace.guaranteed annotation",
+			zap.String("namespace", namespaceObj.Name),
+			zap.String("namespace.guaranteed val is", namespaceGuaranteed))
```

Comment:
`namespace.guaranteed val is` -> just simply `namespace.guaranteed`

Reply:
Addressed in latest PR, thanks @pbacsko .